Tools/MGCB/ConsoleLogger.cs

```diff
@@ -11,13 +11,13 @@ public class ConsoleLogger : ContentBuildLogger
     {
         public override void LogMessage(string message, params object[] messageArgs)
         {
-			Console.WriteLine(message, messageArgs);
+			Console.WriteLine(IndentString + message, messageArgs);
```

Comment:
> We write some exceptions directly to Console

Yeah... that is bad as it means some logging could be lost in some scenarios.  Maybe something to fix in a new PR.